src/interfaces/blog.interfaces.tsx

```diff
@@ -0,0 +1,23 @@
+import { IApiResponse } from './api-response.interface';
+
+export interface IBlog {
+	id: string;
```

Comment:
Still bad. It should align with this class

  public class Blog
  {
      [Key]
      public string Id { get; set; }
      
      public string Title { get; set; }
      
      public string PageName { get; set; }
      
      public string Description { get; set; }

      public string ShortDescription { get; set; }

      public string DefaultImagePath { get; set; }

      public bool IsActive { get; set; } = true;

      public bool IsDraft { get; set; } = false;

      public DateTime DateCreated { get; set; }
  }

Reply:
This is still wrong. These are properties it needs to have

  public class Blog
  {
      [Key]
      public string Id { get; set; }
      
      public string Title { get; set; }
      
      public string PageName { get; set; }
      
      public string Description { get; set; }

      public string ShortDescription { get; set; }

      public string DefaultImagePath { get; set; }

      public bool IsActive { get; set; } = true;

      public bool IsDraft { get; set; } = false;

      public DateTime DateCreated { get; set; }
  }